### PR TITLE
moved toolbar padding to upper level

### PR DESCRIPTION
**Pull request by edewit**
so it works for all tabs

### CODE REVIEW

**Comment by joshuawilson** [comment]
[test]

**Comment by joshuawilson** [comment]
@SMahil please review

**Comment by joshuawilson** [comment]
@edewit @SMahil this change appears to add a second scroll bar when you have more work items then can be displayed on the screen.

File: src/app/app.component.scss [hunk 1]
```diff
@@ -14,3 +14,10 @@
 	border-top: 1px solid $color-pf-blue-300;
 	margin-top: 2%;
 }
+
+.app {
+	padding: em(96) 0 em(66) 0;
+}
```
Comment (SMahil) [inline src/app/app.component.scss]: add height:100% to avoid scroll

**Comment by SMahil** [comment]
@edewit if there is secondary menubar then we need more padding-top.... Basically padding-top should get calculated according the header height

**Comment by edewit** [comment]
is there a constant that we could use?

**Comment by SMahil** [comment]
@edewit No we have to create it

**Comment by joshuawilson** [comment]
[test]

**Comment by joshuawilson** [comment]
@edewit the CI server is showing a rebase error when it builds, and fails. @kbsingh any ideas what that is from?

**Comment by joshuawilson** [comment]
@edewit please try to rebase it again.

**Comment by joshuawilson** [comment]
@edewit something is really wrong with your rebase. If you want we can work out how to fix it later today.

**Comment by edewit** [comment]
closed in favour of https://github.com/almighty/almighty-ui/pull/461

**Pull request closed without merging**